<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VEIDS</class>
 <widget class="QMainWindow" name="VEIDS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Violence Event Identification and Detection System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img/img.qrc">
    <normaloff>:/new/ico/LWCap.ico</normaloff>:/new/ico/LWCap.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 15pt;</string>
        </property>
        <property name="text">
         <string>来源：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="source_comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="source_lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="select_pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt;</string>
        </property>
        <property name="text">
         <string>打开...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="start_pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt;</string>
        </property>
        <property name="text">
         <string>开始</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="stop_pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt;</string>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
        <property name="shortcut">
         <string>Space</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 16pt;</string>
        </property>
        <property name="text">
         <string>检测结果：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 16pt;</string>
        </property>
        <property name="text">
         <string>识别结果：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="7" colspan="3">
       <widget class="QLabel" name="IMAGE">
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>288</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>512</width>
          <height>288</height>
         </size>
        </property>
        <property name="text">
         <string>NO IMAGE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3" rowspan="7" colspan="2">
       <widget class="QLabel" name="IMAGE_2">
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>288</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>512</width>
          <height>288</height>
         </size>
        </property>
        <property name="text">
         <string>NO IMAGE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt;</string>
        </property>
        <property name="text">
         <string>人数：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="number">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt;</string>
        </property>
        <property name="text">
         <string>状态：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="state">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt;</string>
        </property>
        <property name="text">
         <string>正常</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5" colspan="2">
       <widget class="QLabel" name="abnormity_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 131, 69);
font: 75 20pt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="5" colspan="2">
       <widget class="QLabel" name="abnormity_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 131, 69);
font: 75 20pt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="5" colspan="2">
       <widget class="QLabel" name="abnormity_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);
font: 75 20pt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="5" colspan="2">
       <widget class="QLabel" name="abnormity_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);
font: 75 20pt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="5" colspan="2">
       <widget class="QLabel" name="abnormity_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);
font: 75 20pt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img/img.qrc"/>
 </resources>
 <connections/>
</ui>
